components/SearchBar/index.tsx

```diff
@@ -28,27 +28,39 @@ function Component() {
   const Router = useRouter();
   const net = useStore($net);
   const zil_address = useStore($wallet);
-  const user = useStore($user); //@todo learn the difference between useStore & getState to share in the sprint3 file
+  const user = useStore($user);
+  const username = user?.name!;
+  const domain = user?.domain!;
+  const is_admin = useStore($isAdmin);
 
-  const [input, setInput] = useState("");
-  const [thisName, setName] = useState("");
-  const [thisDomain, setDomain] = useState("");
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState("");
+  const [search, setSearch] = useState("");
+
+  const spinner = (
+    <i className="fa fa-lg fa-spin fa-circle-notch" aria-hidden="true"></i>
+  );
 
   // Resolve URL
 
   const checkPath = () => {
     const input = window.location.pathname.replace("/", "").toLowerCase();
     if (input === "") {
       return false;
+    } else if (
+      input === "buynftusername" ||
```

Comment:
Shouldn't this be just "buy"?

Reply:
@DevPH-Wistkey

Reply:
Actually it is not needed anymore, thanks for remind me